.github/workflows/upgrade-sandbox-packages.yml

```diff
@@ -0,0 +1,33 @@
+name: Upgrade sandbox packages
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 12 * * 1
+
+jobs:
+  upgrade:
+    permissions:
+      pull-requests: write
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - name: Upgrade sandbox packages
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          npm i @gravity-ui/uikit@latest @gravity-ui/components@latest @gravity-ui/date-components@latest @gravity-ui/icons
+          git config user.email ""
+          git config user.name "Upgrade sandbox packages action"
```

Comment:
Do you want to shape message according to `commitlint` rules?

Reply:
Commit message changed to `chore(sandbox): upgrade sandbox packages`